taccsite_cms/static/site_cms/css/src/_imports/elements/mailto-link.css

```diff
@@ -0,0 +1,7 @@
+/* Allow supporting `u-mailto-text-replace` on all hyperlinks */
+
+@import url("@tacc/core-styles/src/lib/_imports/tools/x-mailto-text-replace.css");
+
+a[data-user][data-domain] {
+  @extend %x-mailto-text-replace;
+}
```

Comment:
This is why https://github.com/TACC/Core-Styles/pull/44 was made, to provide that mixin this imports.

By extending this code, CMS can decide what selector to use, thus I can apply the style to all suitable hyperlinks instead of requiring an editor to add a class to every hyperlink.